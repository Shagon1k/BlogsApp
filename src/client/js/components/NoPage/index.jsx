import React from 'react';
import './styles.scss';

const NoPage = () => (
	<div>
		No such Page :(
	</div>
)

export default NoPage;