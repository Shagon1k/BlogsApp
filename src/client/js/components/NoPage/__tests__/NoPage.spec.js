import React from 'react';
import NoPage from '../index.jsx';

describe('NoPage component', () => {
	it('renders "No such page" correctly', () => {
		const wrapper = shallow(<NoPage/>);
		expect(wrapper).toMatchSnapshot();
	});
	
})