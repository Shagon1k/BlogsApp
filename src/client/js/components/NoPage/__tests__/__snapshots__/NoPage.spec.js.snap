// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPage component renders "No such page" correctly 1`] = `
<div>
  No such Page :(
</div>
`;
